### PR TITLE
Implement wrappers for GcRule.

### PR DESCRIPTION
**Pull request by coryan**
This is part of the fixes for #79.  Introduce wrappers for GcRule, they will be needed to implement CreateTable().  The

### CODE REVIEW

File: bigtable/admin/column_family.h [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+  }
+
+  /**
+   * Return a garbage collection rule that deletes cells in a column older than
+   * the given duration.
+   *
+   * The function accepts any instantiation of std::chrono::duration<> for the
+   * @p duration parameter.
+   *
+   * @tparam Rep the Rep tparam for @p duration type.
```
Comment (mbrukman) [inline bigtable/admin/column_family.h]: This is exactly what it **is**, but it doesn't tell me what it _means_ or what it _does_.

Same on the next line.

Comment (coryan) [reply bigtable/admin/column_family.h]: Fixed.

Also fixed another place in the code where the comments were similarly vacuous.

**Comment by coryan** [comment]
PTAL.

File: bigtable/admin/column_family.h [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+  }
+
+  /**
+   * Return a garbage collection rule that deletes cells in a column older than
+   * the given duration.
+   *
+   * The function accepts any instantiation of std::chrono::duration<> for the
+   * @p duration parameter.
+   *
+   * @tparam Rep a placeholder to match the Rep tparam for @p duration type, the
```
Comment (garye) [inline bigtable/admin/column_family.h]: I find all of this explanation confusing but it's actually not hard to use. Maybe lead with an example for these?

File: bigtable/admin/column_family.h [hunk 3]
```diff
@@ -47,8 +47,16 @@ class GcRule {
    * The function accepts any instantiation of std::chrono::duration<> for the
    * @p duration parameter.
    *
-   * @tparam Rep the Rep tparam for @p duration type.
-   * @tparam Period the Period tparam for @p duration type.
+   * @tparam Rep a placeholder to match the Rep tparam for @p duration type, the
+   *     semantics of this template parameter are documented in
+   *     std::chrono::duration<> (in brief, the underlying arithmetic type used
```
[the remark this answers is not in the document]
Comment (coryan) [reply bigtable/admin/column_family.h]: Done in #112

**Pull request merged**